cmd/manager/main.go

```diff
@@ -67,6 +63,35 @@ func main() {
 	// uniform and structured logs.
 	logf.SetLogger(zap.Logger())
 
+	viper.BindPFlags(pflag.CommandLine)
+	viper.AutomaticEnv()
+	viper.SetEnvPrefix(viper.GetString("env.prefix"))
+	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
+	viper.AddConfigPath(viper.GetString("config.source"))
+	if viper.GetString("config.file") != "" {
+		viper.SetConfigName(viper.GetString("config.file"))
+		if err := viper.ReadInConfig(); err != nil {
+			//log.Fatalf("cannot load configuration: %v", err)
+			log.Error(err, "cannot load configuration")
+		}
+	}
+
+	// Check to see if the required configuration arguments are present
+	if len(viper.GetString("aws.access.key.id")) == 0 {
```

Comment:
When the operator is ran in a real kubernetes cluster, the AWS access key and access key ID will not be exported with it as it using existing keys on the cluster. I think this check, and the one for the key ID, would fail when starting up while running as a container so we would need to check the development variable to see if these should be set or not

Reply:
We need some way to check whether the AWS secret keys are present when we set-up the cluster. How do we do that?

Reply:
I think we could look at the ClusterOperatorDevelopment flag first to determine if we are running locally or not If thats true, then we want to check for the AWS creds and export them as you're doing. If not, then we can ignore the check and assume we are running on a cluster with a role. When on a cluster, there won't be explicit AWS creds exported to the container so I don't think we can check that properly. If there was no role on the cluster or it didn't have sufficient permissions, that would be an error we expect to see when deploying and would be resolved on the cluster side, not the application side